storybookjs/storybook
# Core: Add STORYBOOK environment variable

Issue: #12416

## What I did
By Default, the environment variable `STORYBOOK` is set to `true`.

## How to test

- Is this testable with Jest or Chromatic screenshots?
yes.
I have tested that `process.env.STORYBOOK` is set to true.
![image](https://user-images.githubusercontent.com/32378535/97977920-3dcbab80-1e10-11eb-828b-745c8b5a4998.png)

- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?
Should I update [this document](https://storybook.js.org/docs/react/configure/environment-variables)?
